net/instaweb/rewriter/rewrite_options.cc

```diff
@@ -995,6 +996,9 @@ std::vector<DeprecatedOptionMap> kDeprecatedOptionNameList(
     kDeprecatedOptionNameData + arraysize(kDeprecatedOptionNameData)
 );
 
+int kNonAddableResourceHeadersSize = 12;
+StringPieceVector kNonAddableResourceHeaders(kNonAddableResourceHeadersSize);
```

Comment:
@jmarantz Out of curiosity, is kDeprecatedOptionNameList (declared one line above) valid according to the style guide? That line was why I thought that what I had initially was OK.